.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
We can't just ignore /bin since cake.* files are in there. Should we just unignore those 3 files? Does it matter? Are there any chances of conflict?